.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build
+
+on:
+  push:
+    branches: main
+  # This will eventually be run on a schedule, daily initially, but some might be better weekly.
+  #  schedule:
+  #    - cron: '0 0 * * *'
```

Comment:
```suggestion
    schedule:
       - cron: '0 0 * * *'
```

Reply:
This enables running daily at midnight, IINM.

github doesn't recognise @daily, @weekly, etc aliases.